code/UMLModel.py

```diff
@@ -135,20 +135,22 @@ def rename_attribute(self, class_name:str, old_attr_name:str, new_attr_name:str)
         # checks if the class exists
         if class_name not in self.classes:
             print("{} does not exist.".format(class_name))
+            return
 
         # checks if the attribute exists in the class
-        elif old_attr_name not in self.classes[class_name].attributes:
+        if not self.classes[class_name].has_attribute(old_attr_name):
             print("{} does not exist in {}.".format(old_attr_name, class_name))
+            return
         
         # checks if the inputted new attribute name already exists in the class
-        elif new_attr_name in self.classes[class_name].attributes:    
+        if self.classes[class_name].has_attribute(new_attr_name):  
             print("{} already exists in {}".format(new_attr_name, class_name))
+            return
```

Comment:
I changed error cases to be single if statements and to return. I feel like this is cleaner than having a bunch of nested if statements that you have to sift through to find the actual modifying code.